code-challenges.js

```diff
@@ -64,7 +168,30 @@ var refactorTester3 = "IAmACodingMaster"
 var utensils = ["fork", "knife", "tongs", "chopsticks", "skewer"]
 var vessels = ["cup", "mug", "stemware", "glass", "tumbler", "tea cup"]
 
-
+// Create a function that takes in 1 argument which is an array of strings
+const mostChars = (array) => {
+    // Create a variable that will store the string with the most characters
+    let strMostChars;
+    // Create a variable that will store the length of the longest string
+    let longestLength = 0;
+    // We will iterate through each element of the array, checking for the value's length
+    for (let i = 0; i < array.length; i++) {
```

Comment:
there is opportunity to use `.forEach((value, index) => {...})` to leverage the higher order function and index